PublicMapping/districtbuilder-classic
# Check in settings.py

## Overview

Check `settings.py` in so we don't need to worry about copying it over when settings change.

### Notes

Not doing anything with `config.xml` for now.

## Testing Instructions

* `docker-compose build && docker-compose up`
* Run `vagrant rsync-auto` separately if not already running

The app should come up normally and any changes to `settings.py` should show up in the VM and in source control.
